src/sdk/model/location/vm-configuration-limits.ts

```diff
@@ -0,0 +1,63 @@
+import { VmConfigurationLimitsJson } from './__json__/vm-configuration-limits-json';
+
+export class VmConfigurationLimits {
```

Comment:
did you not autogenerate this @ariquinones ?

Reply:
@cfsnyder okay i just updated with the auto generator